pkgs/development/python-modules/ansible/default.nix

```diff
@@ -29,6 +30,10 @@ buildPythonPackage rec {
 
   prePatch = ''
     sed -i "s,/usr/,$out," lib/ansible/constants.py
+
+    # Use the unwrapped version of ansible-connection as ansible executes
+    # it directly through the python interpreter.
+    sed -i "s,ansible-connection,.ansible-connection-wrapped," lib/ansible/executor/task_executor.py
```

Comment:
Removing `python` from
```
[python, ansible_connection, to_text(os.getppid())]
```
is likely more robust. It would also be worth upstreaming that change IMO.

Reply:
This would remove the guarantee that `ansible_connection` gets executed with the same Python version, wouldn't it?

It was like this before until https://github.com/ansible/ansible/commit/0feea669880e4387e7d243812fb8ba9e9d1bbb16 which was necessary to fix https://github.com/ansible/ansible/issues/35902, so I think they'd be reluctant to accept that change.

Reply:
It is indeed not guaranteed that it is executed with the exact same interpreter. That is a lousy requirement anyway, because having the same interpreter does not guarantee that it will have the required packages, which is what their intent is. Instead they should rely on their executables already pointing to the right Python.

Reply:
Is this still an issue for you?